src/ubuntu/18.04/Dockerfile

```diff
@@ -44,3 +43,35 @@ RUN apt-get update \
         uuid-dev \
     && rm -rf /var/lib/apt/lists/*
 
+# Build and install curl 7.45.0
+RUN wget https://curl.haxx.se/download/curl-7.45.0.tar.lzma && \
```

Comment:
I think it would be best to place the `&&` as the beginning of the lines to be consistent with the rest of the Dockerfile.  The `&&` at the front is a common Dockerfile practice.

Reply:
Fixed.